BackEnd/bin/www

```diff
@@ -100,3 +100,5 @@ function onListening() {
         : 'port ' + addr.port;
     debug('Listening on ' + bind);
 }
+
+module.exports = server;
```

Comment:
Could you explain why this is here? I don't fully understand exports.

Reply:
Sure, so this is the file that defines the server and the port that it should listen on and all that. I need to export it so that I can import it in `tests.js` to make requests to the server (see https://github.com/rohan-varma/CS-M117-Project/pull/13/files#diff-0fe41b8347475f8c2b90356eb5c4d4e0R7) . So if it weren't exported here we wouldn't be able to access it anywhere else.